fineract-doc/src/docs/en/chapters/architecture/fineract-database-handling.adoc

```diff
@@ -0,0 +1,114 @@
+= Fineract Database Handling
+
+== Fineract Database Upgrades
+
+Fineract supports multiple databases:
+
+* MySQL compatible databases (e.g. MariaDB)
+* PostgreSQL
+
+The platform differentiates between these database types in certain cases when
+there's a need to use some database specific tooling. To do so, the platform
+examines the JDBC driver used for running the platform and tries to determine
+which database is being used.
+
+Fineract doesn't guarantee a backward compatible database upgrades.
+
+=== Background
+
+The Mifos team has stated that non-backward compatible database upgrades are
```

Comment:
I don't think we should say "The Mifos team". This is a community driven project under the Apache umbrella, not a Mifos driven project.

Reply:
i don't think we should add this doc at all as customer documenation. More of a planning and ideas doc. we need to discuss this on where and why this doc can find a place. cc @edcable.

Reply:
@tapaswenipathak as you have noted and as @jdailey suggested, this documentation shouldn't include any of the rationale based on customer's needs of why the changes were made. You are both correct that this portion of the documentation shouldn't include all of this background which almost seems like a case study. 

Regarding @galovics comment, we should refer to it being a community-driven project under Apache umbrella but to give you context Arnold, these documentation-related efforts are all purely Mifos-driven as part of our Google Season of Docs project that we secured with support from Google. 

Intent of this section were to note the changes that were made to support PostgreSQL and Eclipselink, (to the extent that we can), the changes that needed to be made to support PostgreSQL vs MySQL/MariaDB, the process for developers/implementers to upgrade to benefit from these changes and then to specifically document the tuning and indexing specific to PostgreSQL to achieve optimal performance.